### PR TITLE
refactor: rename db-capacity flag to cache-capacity

### PR DESCRIPTION
**Pull request by anatollupacescu**
This will be merged only after the `beekeeper` PR is merged.

This change is [Reviewable](https://reviewable.io/reviews/ethersphere/bee/1704)

### CODE REVIEW

File: packaging/bee.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ config: /etc/bee/bee.yaml
 ## data directory (default "/home/<user>/.bee")
 data-dir: /var/lib/bee
 ## db capacity in chunks, multiply by 4096 to get approximate capacity in bytes
```
Comment (acud) [inline packaging/bee.yaml:19]: `db cache capacity`

File: packaging/homebrew/bee.yaml [hunk 2]
```diff
@@ -17,7 +17,7 @@ config: /usr/local/etc/swarm-bee/bee.yaml
 ## data directory (default "/home/<user>/.bee")
 data-dir: /usr/local/var/lib/swarm-bee
 ## db capacity in chunks, multiply by 4096 to get approximate capacity in bytes
```
Comment (acud) [inline packaging/homebrew/bee.yaml:19]: same

File: packaging/scoop/bee.yaml [hunk 3]
```diff
@@ -15,7 +15,7 @@ config: ./bee.yaml
 ## data directory (default "/home/<user>/.bee")
 data-dir: ./data
 ## db capacity in chunks, multiply by 4096 to get approximate capacity in bytes
```
Comment (acud) [inline packaging/scoop/bee.yaml:17]: same

**Review by acud — APPROVED** [review]

**Review by aloknerurkar — APPROVED** [review]
> Reviewed 7 of 7 files at r1.
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/1704)* status: all files reviewed, 3 unresolved discussions (waiting on @anatollupacescu, @esadakar, and @mrekucci)

**Review by zelig — APPROVED** [review]
> Reviewed 7 of 7 files at r1.
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/1704)* status: all files reviewed, 3 unresolved discussions (waiting on @anatollupacescu, @esadakar, and @mrekucci)

**Review by mrekucci — CHANGES_REQUESTED** [review]
> Reviewed 7 of 7 files at r1.
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/1704)* status: all files reviewed, 4 unresolved discussions (waiting on @anatollupacescu and @esadakar)

___
*[cmd/bee/cmd/start.go, line 120 at r1](https://reviewable.io/reviews/ethersphere/bee/1704#-M_doALV9fIabPz9ZPQI:-M_doALV9fIabPz9ZPQJ:b-lzxj7g) ([raw file](https://github.com/ethersphere/bee/blob/995af63fd0cc1604ebbc94b8c70cc7df40d3c9f1/cmd/bee/cmd/start.go#L120)):*
> ```Go
> 			b, err := node.NewBee(c.config.GetString(optionNameP2PAddr), signerConfig.address, *signerConfig.publicKey, signerConfig.signer, c.config.GetUint64(optionNameNetworkID), logger, signerConfig.libp2pPrivateKey, signerConfig.pssPrivateKey, node.Options{
> 				DataDir:                  c.config.GetString(optionNameDataDir),
> 				DBCapacity:               c.config.GetUint64(optionNameDBCapacity),
> ```

You forgot to rename the `DBCapacity` option.

**Review by esadakar — APPROVED** [review]
Good to merge, after acud's requests.

**Review by mrekucci — APPROVED** [review]
> Reviewed 8 of 8 files at r2.
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/1704)* status: all files reviewed, 3 unresolved discussions (waiting on @anatollupacescu)

**Review by anatollupacescu — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/1704)* status: all files reviewed, 3 unresolved discussions (waiting on @acud)

___
*[packaging/bee.yaml, line 19 at r1](https://reviewable.io/reviews/ethersphere/bee/1704#-M__gH74WeCPugl21axk-r1-19:-M_e8XrABDG7YdOrbLo3:b-896fix) ([raw file](https://github.com/ethersphere/bee/blob/fa5b07e38e91753e87ba968b88f0137a9b956bd7/packaging/bee.yaml#L19)):*
<details><summary><i>Previously, acud (acud) wrote…</i></summary><blockquote>

`db cache capacity`
</blockquote></details>

Done.
___
*[packaging/homebrew/bee.yaml, line 19 at r1](https://reviewable.io/reviews/ethersphere/bee/1704#-M__gH74WeCPugl21axl-r1-19:-M_e8Yz37mEWBu_vUFLo:b-896fix) ([raw file](https://github.com/ethersphere/bee/blob/fa5b07e38e91753e87ba968b88f0137a9b956bd7/packaging/homebrew/bee.yaml#L19)):*
<details><summary><i>Previously, acud (acud) wrote…</i></summary><blockquote>

same
</blockquote></details>

Done.
___
*[packaging/scoop/bee.yaml, line 17 at r1](https://reviewable.io/reviews/ethersphere/bee/1704#-M__gH74WeCPugl21axm-r1-17:-M_e8Zeb-WyCdpp7DZq7:b-896fix) ([raw file](https://github.com/ethersphere/bee/blob/fa5b07e38e91753e87ba968b88f0137a9b956bd7/packaging/scoop/bee.yaml#L17)):*
<details><summary><i>Previously, acud (acud) wrote…</i></summary><blockquote>

same
</blockquote></details>

Done.

File: packaging/docker/env [hunk 4]
```diff
@@ -23,7 +23,7 @@ BEE_CLEF_SIGNER_ENABLE=true
 # BEE_CORS_ALLOWED_ORIGINS=[]
 ## data directory (default /home/<user>/.bee)
 # BEE_DATA_DIR=/home/bee/.bee
-## db capacity in chunks, multiply by 4096 to get approximate capacity in bytes
+## cache capacity in chunks, multiply by 4096 to get approximate capacity in bytes
 # BEE_DB_CAPACITY=5000000
```
Comment (vandot) [inline packaging/docker/env:27]: rename `BEE_DB_CAPACITY ` to `BEE_CACHE_CAPACITY`

**Review by anatollupacescu — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/ethersphere/bee/1704)* status: 0 of 12 files reviewed, 4 unresolved discussions (waiting on @acud, @aloknerurkar, @mrekucci, @vandot, and @zelig)

___
*[packaging/docker/env, line 27 at r2](https://reviewable.io/reviews/ethersphere/bee/1704#-M_dxiE9P9Fw_1hGsdeJ-r2-27:-Ma3HOdf9qo2haYAbVID:b-896fix) ([raw file](https://github.com/ethersphere/bee/blob/e2906a86c16177800f3d4415f8af8a56756545c5/packaging/docker/env#L27)):*
<details><summary><i>Previously, vandot (Ivan Vandot) wrote…</i></summary><blockquote>

rename `BEE_DB_CAPACITY ` to `BEE_CACHE_CAPACITY`
</blockquote></details>

Done.

**Pull request merged**